packages/EasyActivity/src/Bridge/Symfony/Resources/config/services.php

```diff
@@ -49,10 +50,13 @@
     $services
         ->alias(BridgeConstantsInterface::SERVICE_SERIALIZER, 'serializer');
 
+    $services->set(BridgeConstantsInterface::SERVICE_CIRCULAR_REFERENCE_HANDLER, CircularReferenceHandler::class);
+
     $services
         ->set(ActivitySubjectDataSerializerInterface::class, SymfonyActivitySubjectDataSerializer::class)
-        ->arg('$disallowedProperties', '%' . BridgeConstantsInterface::PARAM_DISALLOWED_PROPERTIES . '%')
-        ->arg('$serializer', service(BridgeConstantsInterface::SERVICE_SERIALIZER));
+        ->arg('$serializer', service(BridgeConstantsInterface::SERVICE_SERIALIZER))
```

Comment:
If you want this PR to be merged right now, please change `service` to `ref`, then we can fix that file in the coming next version when we drop support for Symfony 4.4

Reply:
Done, replaced by `ref`